frontend/talentsearch/src/js/components/search/SearchForm.tsx

```diff
@@ -152,13 +153,46 @@ const SearchForm = React.forwardRef<SearchFormRef, SearchFormProps>(
       return () => subscription.unsubscribe();
     }, [watch, classificationMap, onUpdateApplicantFilter]);
 
+    const classificationLabels = defineMessages({
```

Comment:
Would these be more stable if they were declared above the FC?  If you moved `getClassificationLabel` above the FC as well, maybe you wouldn't need the useCallback on it?